Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN make
 
 FROM amazonlinux:2
 RUN yum install -y ca-certificates
+RUN yum update -y
```

Comment:
better to be below to save image size :D 
```
RUN yum update -y && \
    yum install -y ca-certificates && \
    yum clean all && \
    rm -rf /var/cache/yum
```

Reply:
Updated it..